src/infrastructure/figma/figma-service.ts

```diff
@@ -344,50 +342,91 @@ export class FigmaService {
 		});
 
 	/**
-	 * Return Atlassian design representation for the given Figma File.
-	 *
-	 * @throws {UnauthorizedFigmaServiceError} Not authorized to access Figma.
+	 * Returns Atlassian design representation for the given Figma File if it exists; otherwise -- `null`.
```

Comment:
minor: Can this method still throw an UnauthorizedFigmaServiceError i.e. if the user is not logged in to Figma? I have a similar question for the getDesignForNode method as well.

Reply:
These private methods currently do not do error translation but now I am thinking that they should :) I've updated these methods and the code comments accordingly. Thank you, @elainefigma , for the careful review!